app/brave_settings_strings.grdp

```diff
@@ -113,6 +113,9 @@
   <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_BRAVE_TAB_HOVER_MODE_TOOLTIP" desc="The label name of the tab hover mode where a tooltip with the site title is shown.">
     Tooltip
   </message>
+  <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_BRAVE_SHARED_PINNED_TAB" desc="The label name of a toggle button for Shared Pinned Tab feature">
+    Shared pinned tabs
```

Comment:
@sangwoo108 can we be a little more descriptive here as to what this toggle would do? For example, `Show pinned tabs in all windows`. cc: @rebron do we have a better wording for this feature? "Shared pinned tabs" as a settings toggle is not immediately obvious, imo.

Reply:
That's much better!